__main__.py

```diff
@@ -1,10 +1,21 @@
 import asyncio
+import logging
 import os
 import sys
 
 import hangups
 
 
+logger = logging.getLogger('hangbot')
+logger.setLevel(logging.DEBUG)  # this could depend on the environment
+
+stdout_handler = logging.StreamHandler()
+stdout_handler.setLevel(logging.DEBUG)
+stdout_handler.setFormatter(
+    logging.Formatter('%(asctime)s - %(name)s - [%(levelname)s] %(message)s'))
```

Comment:
I wasn't really thinking of console usability, since log messages aren't really meant for people to watch during development (they're more for after-the-fact debugging where more information is better).

I did a compromise: I made the console logs terser, but I added in a rotating file handler with the verbose messages. This will be useful when putting the bot into production.